src/app/shared/services/app-config/app-config.service.ts

```diff
@@ -73,8 +73,6 @@ export class AppConfigService extends SyncServiceBase {
    * with the initial config
    */
   public setAppConfig(overrides: IAppConfigOverride = {}) {
-    // Ignore case where no overrides provides or overrides already applied
-    if (Object.keys(overrides).length === 0) return;
```

Comment:
Sometimes, we want to supply an empty override object in order to reset the appConfig to have its initial values.

In the case of this current PR, this happens when the active theme contains no overrides and we navigate from a template which does have template-level app config overrides to one which does not. In that case, the `overrides` passed to this function, representing the result of merging the active theme's overrides with the template overrides, is an empty object. We want setting it here to have the effect of removing any outstanding overrides that have been applied on top of the initial config values, which in this case will be those overrides applied by the template that we are navigating away from.